src/main/kotlin/sg/com/quantai/etl/services/TimescaleDBTableInitializer.kt

```diff
@@ -0,0 +1,42 @@
+package sg.com.quantai.etl.services
+
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.boot.CommandLineRunner
+import org.springframework.context.annotation.Bean
+import org.springframework.jdbc.core.JdbcTemplate
+import org.springframework.stereotype.Component
+
+@Component
+class TimescaleDBTableInitializer(val jdbcTemplate: JdbcTemplate) {
+
+    private val logger: Logger = LoggerFactory.getLogger(TimescaleDBTableInitializer::class.java)
+
+    @Bean
+    fun runDatabaseInitializer(): CommandLineRunner {
+        return CommandLineRunner {
+            // Create Raw Crypto Data Table
+            jdbcTemplate.execute("""
+                CREATE TABLE IF NOT EXISTS raw_crypto_data (
+                    id SERIAL PRIMARY KEY,
+                    symbol VARCHAR(10) NOT NULL,
+                    source VARCHAR(50) NOT NULL,
+                    price DECIMAL NOT NULL,
+                    timestamp TIMESTAMPTZ NOT NULL
+                );
+            """)
+            logger.info("Table 'raw_crypto_data' initialized.")
+
```

Comment:
Use logger to yield an info message before initializing the tables as well

Reply:
done